File: challenges/codeeval/medium/015/lnavarrete.java
package endian;
import java.nio.ByteOrder;
public class Endian 
{
    public static void main(String[] args) 
    {
     if (ByteOrder.nativeOrder().equals(ByteOrder.BIG_ENDIAN)) 
     {
  System.out.println("BigEndian");
     } else 
     {
  System.out.println("LittleEndian");
    }
    }
}

File: challenges/codeeval/medium/015/lnavarrete.cs
using System;
namespace Endian
{
    class Program
    {
        static void Main(string[] args)
        {
       
        if (BitConverter.IsLittleEndian)
            Console.WriteLine("LittleEndian");
        else Console.WriteLine("BigEndian");
	Console.ReadLine(); 
        }
    }
}
